md/single-sign-on-with-oidc.md

```diff
@@ -0,0 +1,316 @@
+:::
+
+## Configure logout behaviour
+
+If your Bonita platform is configured to manage authentication over OIDC, when users log out of Bonita Portal, they do not log out of the OIDC Identity Provider (IdP).
+Therefore they are not logged out of all applications that are using the OIDC provider.
+To avoid this, you have two options :
+
+#### Hide the logout button of the portal
+
+This is the most commonly used solution. Users are logged in as long as they don't close their web browser (unless their session times out).
+To do this, set the `logout.link.hidden` option to `true` in `authenticationManager-config.properties` located in `<BUNDLE_HOME>/setup/platform_conf/initial/tenant_template_portal` for not initialized platform or `<BUNDLE_HOME>/setup/platform_conf/current/tenant_template_portal` and `<BUNDLE_HOME>/setup/platform_conf/current/tenants/[TENANT_ID]/tenant_portal/`.
+
+::: info
+**Note:** When a user logs out from the IdP directly, Bonita Portal's session will remain active. The user's session time to live will be reset
+to the configured session timeout value upon each user interaction with the server.
+:::
+
+#### Setup Bonita platform for OIDC logout
+
+Global logout allows to log out from the OIDC provider as well as all the registered Service Providers when logging out from Bonita platform. This is sometimes required for example if users are on public computers.
+As OIDC Providers do not necessarily support single logout and have different ways of handling it (there are several OIDC Single Logout methods), Bonita only offers OIDC logout through an OIDC logout URL that the IpP should provide and support.
+To setup Bonita for OIDC logout:
+1. Set the value of the property `oidc.logout.url` with your OIDC provider logout URL in `authenticationManager-config.properties` located in `<BUNDLE_HOME>/setup/platform_conf/initial/tenant_template_portal` for not initialized platform or `<BUNDLE_HOME>/setup/platform_conf/current/tenant_template_portal` and `<BUNDLE_HOME>/setup/platform_conf/current/tenants/<TENANT_ID>/tenant_portal/`.
+2. If the logout URL of your OIDC provider supports a parameter to redirect after the logout, you can set the property `oidc.logout.url.redirectQuery` with the query string to use (it will be concatenated to the logout URL). The placeholder `{encodedRedirectUri}` will be replaced at runtime with the current Bonita URL at the time on the logout action.
+
+::: info
+**Note:** If the logout flow supported by your OIDC provider is not the same as the one supported by Bonita platform, the preferred solution to handle it anyway is to intercept the requests to /logoutService and handle the logout programmatically.
+:::
+
+## Troubleshoot
+
+To troubleshoot SSO login issues, you need to increase the [log level](logging.md) to `ALL` for the packages `org.bonitasoft`, `com.bonitasoft`, and `org.keycloak` in order for errors to be displayed in the log files bonita-*.log (by default, they are not).
```

Comment:
Are we sure we want even FINEST trace even on the whole `org.bonitasoft`, `com.bonitasoft` ? that's a lot

Reply:
FINE is enough for bonita
For keycloak, I prefer ALL. I will update the page